### DISCUSSION TITLE
Is it possible to quit wezterm without asking?

### DISCUSSION THREAD

**Discussion by Demianeen** [wezterm/wezterm#5189] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### Which Wayland compositor or X11 Window manager(s) are you using?

Yabai

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Generally, the same question as in the question above. Is there any option that I can set to quit wezterm without asking to confirm? I searched through docs and the only thing that I found is QuitApplication action, but I get an error if I pass parameters there that it's a string.

**Comment by nekowinston** [comment, marked as answer]
Search the docs for "close confirmation", this config setting should work: https://wezfurlong.org/wezterm/config/lua/config/window_close_confirmation.html

**Reply by Demianeen** [reply]
I searched for close confirmation. It only appears in my case when I search "window close confirmation")

I would never figure out by myself, @nekowinston. Thanks!
